pkgs/development/web/mailcatcher/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "mailcatcher-${version}";
+
+  version = (import ./gemset.nix).mailcatcher.version;
+
+  env = bundlerEnv {
+    name = "${name}-gems";
+
+    inherit ruby;
+
+    gemdir = ./.;
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = ["installPhase"];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    makeWrapper ${env}/bin/mailcatcher $out/bin/mailcatcher
+    makeWrapper ${env}/bin/catchmail $out/bin/catchmail
+  '';
+
+  meta = with lib; {
+    description = "Catches mail and serves it through a dream";
```

Comment:
At first I wrote "Local SMTP server that shows emails in a web interface", then I looked for other distros' description but the only thing I could find was the AUR, which uses the upstream description. Also I think that "serves it through a dream" is not really helpful for search.
I would like to find a better wording for "Local SMTP server that shows emails in a web interface", any suggestion?